mappings/net/minecraft/client/util/math/PosAndRot.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4462 net/minecraft/client/util/math/PosAndRot
```

Comment:
That's a bad name

Reply:
It's a Mojang name. See the class' `toString`.